module/proto/gravity/v1/attestation.proto

```diff
@@ -39,7 +29,9 @@ message Attestation {
   bool                observed = 1;
   repeated string     votes    = 2;
   uint64              height   = 3;
-  google.protobuf.Any claim    = 4;
+  google.protobuf.Any claim    = 4 [
+    (cosmos_proto.accepts_interface) = "EthereumClaim"
```

Comment:
I haven't seen this option work

Reply:
This is how the sdk handles any with gov and keys.

Reply:
I've seen it working fwiw